src/dicomweb_client/cli.py

```diff
@@ -62,6 +62,10 @@ def _get_parser():
         '--url', dest='url', metavar='URL',
         help='uniform resource locator of the DICOMweb service'
     )
+    parser.add_argument(
+        '--chunk_size', dest='chunk_size', metavar='CHUNK_SIZE',
```

Comment:
Could you please change the value of ``metavar`` from ``"CHUNK_SIZE"`` to ``"NUM"`` to follow the same logic as the other arguments.